data/json/furniture_and_terrain/furniture-medical.json

```diff
@@ -405,13 +405,13 @@
     "type": "furniture",
     "id": "f_HPLC",
     "name": "high performance liquid chromatographer",
+    "looks_like": "f_chemical_mixer",
     "description": "This high-tech tool would, with electricity and an experienced user, be a very useful way to separate chemicals in a liquid or aqueous phase based on their affinity to a solid state medium in a tube.  In other words, it's a fancy way to separate things.",
     "symbol": ":",
     "color": "red_white",
     "move_cost_mod": -1,
     "coverage": 40,
     "required_str": 12,
-    "looks_like": "t_machinery_light",
```

Comment:
I don't recommend changing existing looks_likes without cause: you've now made three different pieces of furniture all look the same if they don't have specifically drawn tiles.